scripts/create_test_data.py

```diff
@@ -194,6 +200,63 @@ def create_test_data(db: orm.Session) -> FidesUser:
                     phone_number="+1 234 567 8910",
                 ),
             )
+            for action_type in [
+                AuditLogAction.denied,  # Run denied before approved to simulate a realistic order
+                AuditLogAction.approved,
+                AuditLogAction.finished,
+            ]:
+                AuditLog.create(
+                    db=session,
+                    data={
+                        "user_id": "system",
+                        "privacy_request_id": pr.id,
+                        "action": action_type,
+                        "message": f"Audit log for request with id {pr.id} and action_type {action_type}",
+                    },
+                )
+
+            for action_type in [
+                ActionType.access.value,
+                ActionType.erasure.value,
+            ]:
+                for exl_status in [
+                    ExecutionLogStatus.in_processing,
+                    ExecutionLogStatus.pending,
+                    ExecutionLogStatus.complete,
+                    ExecutionLogStatus.error,
+                    ExecutionLogStatus.paused,
+                    ExecutionLogStatus.retrying,
+                    ExecutionLogStatus.skipped,
+                ]:
+                    ExecutionLog.create(
+                        db=db,
+                        data={
+                            "dataset_name": "dummy_dataset",
+                            "collection_name": "dummy_collection",
+                            "fields_affected": [
+                                {
+                                    "path": "dummy_dataset:dummy_collection:dummy_field_1",
+                                    "field_name": "dummy_field",
+                                    "data_categories": [
+                                        "data_category_1",
```

Comment:
I think this is a fine for now, but at some point we may be using this data and have validation where Pydantic expects them to be proper data categories.